src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.approved.txt

```diff
@@ -210,6 +210,10 @@ namespace GraphQL
         protected virtual GraphQL.Language.AST.Operation GetOperation(string operationName, GraphQL.Language.AST.Document document) { }
         protected virtual GraphQL.Execution.IExecutionStrategy SelectExecutionStrategy(GraphQL.Execution.ExecutionContext context) { }
     }
+    public static class DocumentWriterExtensions
```

Comment:
By the way, add new package to api approval test :)
```c#
  [InlineData(typeof(DocumentWriter))]
```
in ApiApprovalTests.cs

Reply:
I created it as a separate unit test since adding it to the existing PublicApi test caused it to fail as the approval text file can only match only one of the assemblies.

Reply:
OK. You can use overload with an additional parameter.

Reply:
Merged the two unit tests together now, using the assembly name as the discriminator